app/cdap/components/hydrator/components/TopPanel/NameAndDescription.tsx

```diff
@@ -74,17 +79,71 @@ const CancelButton = styled(Button)`
   text-transform: none;
 `;
 
-const ButtonsContainer = styled.div`
-  float: left !important;
+const StyledIconSVG = styled(IconSVG)`
+  display: inline-block;
+  font-size: 20px !important;
+  line-height: 35px;
+  margin-top: 10px;
+`;
+
+const PipelineType = styled.div`
+  display: inline-block;
+  padding-left: 7px;
+  color: #397cf1;
+  text-decoration: none;
+`;
+
+const MetadataLeft = styled.div`
+  margin-left: 10px;
+  width: calc(100% - 50px);
+  width: -moz-calc(100% - 50px);
+  width: -webkit-calc(100% - 50px);
+  text-align: left !important;
+`;
+
+const HydratorMetadata = styled.div`
+  width: calc(100vw - 805px);
+  width: -webkit-calc(100vw - 805px);
+  flex-wrap: wrap;
+  display: flex;
+  background-color: transparent;
+  z-index: 999;
+  box-shadow: none;
+
+  ${({ expanded }) =>
+    expanded &&
+    `background-color: white;
+    height: 185px;
+    cursor: auto;
+    padding: 10px;
+    width: 100%;
+    position: absolute;
+    box-shadow: 0 7px 7px rgb(0 0 0 / 10%), 0 19px 59px rgb(0 0 0 / 20%);`}
+`;
+
+const NameTextField = styled(TextField)`
+  width: 100%;
+  max-width: 100%;
+  .MuiOutlinedInput-input {
+    padding: 10px 14px;
+  }
+`;
+
+const DescriptionTextField = styled(TextField)`
+  width: 100%;
+  max-width: 100%;
+  margin: 12px 0 15px 0;
```

Comment:
I think it'd be better if you moved these styles to their own file like `styles.ts` and just import them here. Helps with readability.